### DISCUSSION TITLE
How can I kill this error msg

### DISCUSSION THREAD

**Discussion by tjgh1322** [Natizyskunk/vscode-sftp#163] [Q&A] [answered]
When I use "SFTP : Remote to Local" this thing occurs
![image](https://user-images.githubusercontent.com/82634713/169730426-17f7de1d-9659-4c94-9b70-8d76650799ac.png)

Of course, there is no problem with the function... But everyday my companies ask me about this things and I always say 'Ignore it'
But I want to solve it

and the error.log is like this
![image](https://user-images.githubusercontent.com/82634713/169730914-bf98dce9-6fe2-41e3-a038-0114d1231bc3.png)

**Comment by Natizyskunk** [comment, marked as answer]
There could be multiple reasons :
1. It's a temp file that is not available anymore when sftp try to mull it on your local.
2. It's a bug related to the rights on the file (or parent folder).
3. The file simply doesn't exist.
